apps/veil/src/pages/tournament/api/use-personal-rewards.ts

```diff
@@ -69,8 +70,7 @@ const fetchRewards = async (
       '/api/tournament/delegator-history',
       {
         epochs: Array.from(epochs),
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- address is defined
-        address: { inner: Array.from(address!.inner) },
+        address: address ? bech32mAddress(address) : '',
```

Comment:
I think we should throw here, or explicitly mark this function as being able to return undefined, because otherwise we'll just get an error when the server sees an empty string and doesn't like that.

Reply:
the endpoint throws 400 in this rare case, so it's handled

Reply:
Yeah, I guess it's fine to defer to the endpoint